.github/workflows/merge-conflict-labeler.yaml

```
@@ -8,10 +8,10 @@ on:
 
 jobs:
   triage:
-    runs-on: ubuntu-20.04
+    runs-on: ubuntu-latest
```

```suggestion
    runs-on: ubuntu-20.04
```

Please keep the version specific image